core/makenek.inc

```diff
@@ -251,6 +250,7 @@ if [ "$IFCMT" == "true" -o "$IFCMT" == "yes" ]; then
 else
   # default
   IFCMT=false
+  PPLIST="${PPLIST} MPI"
```

Comment:
To be more precise.  Nek5000:release/v16.0.0 includes the fix from PR #81, but stgetke:feature/SIZE clobbered that fix.